### PR TITLE
Aggregate service cache clean up.

### PR DESCRIPTION
**Pull request by agorina**

### CODE REVIEW

File: aggregate-service/src/main/java/org/opentestsystem/rdw/olap/CacheConfiguration.java [hunk 1]
```diff
@@ -46,66 +28,16 @@
...
     @Scheduled(cron = "${app.cache.repository.refresh-cron}", zone = "GMT")
     @CacheEvict(cacheNames = {
             Organizations,
             AdministrationConditions,
+            ActiveAssessment,
             Assessments,
             AssessmentGrades,
-            AssessmentTypes,
-            Booleans,
-            StrictBooleans,
```
Comment (agorina) [inline aggregate-service/src/main/java/org/opentestsystem/rdw/olap/CacheConfiguration.java]: These are not changed by the migrate, so no need to clear them from the cache.

File: aggregate-service/src/main/java/org/opentestsystem/rdw/olap/CacheConfiguration.java [hunk 2]
```diff
@@ -46,66 +28,16 @@
...
-
     /**
-     * Refreshes the contents of the report organizations cache
+     * Clear the contents of the cache
      */
     @Scheduled(cron = "${app.cache.repository.refresh-cron}", zone = "GMT")
     @CacheEvict(cacheNames = {
             Organizations,
             AdministrationConditions,
+            ActiveAssessment,
```
Comment (agorina) [inline aggregate-service/src/main/java/org/opentestsystem/rdw/olap/CacheConfiguration.java]: This was missing.

File: aggregate-service/src/main/java/org/opentestsystem/rdw/olap/CacheConfiguration.java [hunk 3]
```diff
@@ -115,29 +47,9 @@
             Genders,
             SchoolYears,
             Subjects,
-            Embargo
+            Embargo,
+            ReportOptions
```
Comment (agorina) [inline aggregate-service/src/main/java/org/opentestsystem/rdw/olap/CacheConfiguration.java]: and this was missing too.

File: aggregate-service/src/main/java/org/opentestsystem/rdw/olap/CacheConfiguration.java [hunk 4]
```diff
@@ -115,29 +47,9 @@
...
-        embargoRepository.findAllEmbargoedDistrictIds();
-        ethnicityRepository.findAll();
-        genderRepository.findAll();
-        organizationRepository.findMapOfDistrictsByIds();
-        organizationRepository.findMapOfSchoolsById();
-        organizationRepository.findAllSchoolsByDistrictId();
-        schoolYearRepository.findAll();
-        strictBooleanRepository.findAll();
-        subjectRepository.findAll();
-        targetRepository.findAll();
```
Comment (agorina) [inline aggregate-service/src/main/java/org/opentestsystem/rdw/olap/CacheConfiguration.java]: We do not want to refresh the cache.

File: aggregate-service/src/main/java/org/opentestsystem/rdw/olap/repository/impl/JdbcAdministrationConditionRepository.java [hunk 5]
```diff
@@ -19,7 +19,7 @@
         super(template, findAllQuery);
     }
 
-    @Cacheable(AggregateServiceCache.AdministrationConditions)
+    @Cacheable(value = AggregateServiceCache.AdministrationConditions, key = "#root.methodName")
```
Comment (agorina) [inline aggregate-service/src/main/java/org/opentestsystem/rdw/olap/repository/impl/JdbcAdministrationConditionRepository.java]: I was trying to write a test for this and had a hard time retrieving the data by the 'default' key. Then I ran across recommendations that it is considered a better practice to explicitly define the key.

File: aggregate-service/src/main/resources/application.yml [hunk 6]
```diff
@@ -61,7 +61,7 @@ server:
 spring:
   cache:
     type: SIMPLE
-    cache-names: internalTranslationsByCode, reportOptions, administrationConditions, assessments, assessmentGrades, assessmentTypes, booleans, strictBooleans, organizations, completenessCodes, claims, targets, ethnicities, genders, elases, schoolYears, subjects, activeAssessments, embargo
```
Comment (agorina) [inline aggregate-service/src/main/resources/application.yml]: `internalTranslationsByCode` is not used.

File: aggregate-service/src/main/java/org/opentestsystem/rdw/olap/repository/impl/JdbcOrganizationRepository.java [hunk 7]
```diff
@@ -71,7 +71,6 @@
         }
     }
 
-
```
Comment (jtreuting) [inline aggregate-service/src/main/java/org/opentestsystem/rdw/olap/repository/impl/JdbcOrganizationRepository.java]: this was for @malaffoon

File: aggregate-service/src/main/java/org/opentestsystem/rdw/olap/repository/impl/JdbcAdministrationConditionRepository.java (hunk 5 above)
Comment (wtritch) [reply aggregate-service/src/main/java/org/opentestsystem/rdw/olap/repository/impl/JdbcAdministrationConditionRepository.java]: TIL

File: aggregate-service/src/main/java/org/opentestsystem/rdw/olap/repository/impl/JdbcOrganizationRepository.java (hunk 7 above)
Comment (agorina) [reply aggregate-service/src/main/java/org/opentestsystem/rdw/olap/repository/impl/JdbcOrganizationRepository.java]: In this case I am going to produce more of this and in exchange I am expecting a chapter dedicate to @jtreuting and me.

**Pull request merged**